<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpServerWidget</class>
 <widget class="QWidget" name="TcpServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="1">
    <widget class="QWidget" name="widget_input" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>250</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_sendPanel" native="true">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>250</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_output" native="true"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>串口设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="11">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>已连接客户端</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLineEdit" name="lineEdit_localPort">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>1314</string>
        </property>
       </widget>
      </item>
      <item row="0" column="13">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="6">
       <widget class="QComboBox" name="comboBox_localAddress"/>
      </item>
      <item row="0" column="10">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>服务器地址</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>服务器端口</string>
        </property>
       </widget>
      </item>
      <item row="0" column="14">
       <widget class="QPushButton" name="pushButton_open">
        <property name="text">
         <string>开启服务器</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QComboBox" name="comboBox_client">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>无客户端连接</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
